src/Razor/test/testapps/PackageLibraryDirectDependency/PackageLibraryDirectDependency.csproj

```diff
@@ -31,6 +31,10 @@
   <ItemGroup Condition="'$(BinariesRoot)'!=''">
     <Reference Include="$(BinariesRoot)\Microsoft.AspNetCore.Razor.Test.ComponentShim.dll"/>
     <Reference Include="$(BinariesRoot)\Microsoft.AspNetCore.Razor.Test.MvcShim.ClassLib.dll"/>
+
+    <!-- We don't need anything in these assemblies, we just want to make sure they are built -->
+    <ProjectReference Include="..\..\..\src\Microsoft.NET.Sdk.Razor\Microsoft.NET.Sdk.Razor.csproj" ReferenceOutputAssembly="false" />
```

Comment:
Hmm - do the other projects have this?

Reply:
No, i'm trying a different approach to make this work. Sorry for the noise